api/src/domain/environment.rs

```diff
@@ -15,4 +15,4 @@ pub struct EnvironmentValue {
     pub enabled: bool,
 }
 
-unsafe impl Send for EnvironmentFile{}
+unsafe impl Send for EnvironmentFile {}
```

Comment:
what makes this unsafe?

Reply:
just curious if you think it can be rewritten to not be unsafe at some point or if this is to get around some edge case

Reply:
Im not entirely sure, this was leftover from something I was trying to make this struct work for the env var table. Was complaining that the struct didn't implement Send or Sync to be async, and then it complain the impl wasn't marked `unsafe`?